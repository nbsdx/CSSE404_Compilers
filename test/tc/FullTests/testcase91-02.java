//invalid return types
class boring
{
   public static void main( String[] args )
   {
      System.out.println(0);
   }
}

class UV {
	int a;
	public int method(int test){
		return true;
	}
}

class shadow extends UV{
	int b;
	public int method(int test){
		return 0;
	}
}