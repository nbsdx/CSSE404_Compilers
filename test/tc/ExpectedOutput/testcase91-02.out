Actual return type boolean of method method does not match declared type int
